pkg/kubelet/config/config.go

```diff
@@ -311,6 +314,9 @@ func (s *podStorage) merge(source string, change interface{}) (adds, updates, de
 		}
 	case kubetypes.RESTORE:
 		glog.V(4).Infof("Restoring pods for source %s", source)
+		for _, value := range update.Pods {
```

Comment:
had to add this because restores were not propogated through to the updates channel

Reply:
make sense